contentcuration/contentcuration/frontend/shared/data/resources.js

```diff
@@ -1193,6 +1202,26 @@ export const ContentNode = new TreeResource({
       return this.requestCollection({ ancestors_of: id });
     });
   },
+
+  /**
+   * Uses local IndexedDB index on node_id+channel_id, otherwise specifically requests the
+   * collection using the same params since GET detail endpoint doesn't support that the params
+   *
+   * @param {String} nodeId
+   * @param {String} channelId
+   * @return {Promise<{}|null>}
+   */
+  getByNodeIdChannelId(nodeId, channelId) {
+    const values = [nodeId, channelId];
+    return this.table.get({ '[node_id+channel_id]': values }).then(node => {
+      if (!node) {
+        return this.requestCollection({ _node_id_channel_id___in: values.join(',') }).then(
```

Comment:
I don't think you should need to do the join here - the get parameter serialization should handle this part.

For consistency, setting values to `[[nodeId, channelId]]` because we're doing an `__in` query is cleanest, although I think it has the same net result.